lib/vm/src/instance/mod.rs

```diff
@@ -1037,6 +1037,27 @@ impl InstanceHandle {
         Ok(())
     }
 
+    /// Similar to finish_instantiation, except it doesn't invoke start function.
+    pub unsafe fn finish_instantiation_without_start_func(
+        &self,
+        data_initializers: &[DataInitializer<'_>],
+    ) -> Result<(), Trap> {
+        let instance = self.instance().as_ref();
+
+        // Apply the initializers.
+        initialize_tables(instance)?;
+        initialize_memories(instance, data_initializers)?;
+
+        Ok(())
+    }
+
+    /// Call start_func on instance
+    pub fn call_start_func(&self, trap_handler: &dyn TrapHandler) -> Result<(), Trap> {
```

Comment:
Let's keep the naming consistent. If we already have `invoke_start_function`, it's better to stick to this terminology instead of adding `call_start_func` synonym.